lld/test/MachO/compact-unwind-foldings.s

```diff
@@ -23,8 +23,8 @@
 ## Check that [1] offset starts at c's address + 3 (its length).
 # CHECK-LABEL: Contents of __unwind_info section:
 # CHECK:  Top level indices: (count = 2)
-# CHECK-NEXT : [0]: function offset=[[#%#.7x,MAIN_ADDR]]
-# CHECK-NEXT : [1]: function offset=[[#%#.7x,C_ADDR + 3]]
+# CHECK-NEXT: [0]: function offset=[[#%#.7x,MAIN_ADDR]]
```

Comment:
@oontvoo @int3 This needs a fix

Reply:
Sorry, I'm not sure I understand this change. Why is line 27 being deleted? From the PR's description seems to say it's only fixing typo ....

Reply:
See full diff https://github.com/llvm/llvm-project/pull/93471/files#diff-e7fc4c1ac531d5a509bc6223274d720c04764fffe99cb7011c8426eb1b2cca25

Reply:
Ah, i see - feel free to revert this file (or temporarily disable the test), I'll have a look and fix it later this week. Thanks!